.github/workflows/pull-request-checks.yaml

```diff
@@ -52,22 +51,21 @@ jobs:
           # user input
           DEBIAN_FRONTEND: noninteractive
         run: |
-          sudo apt-get install -yq cmake ninja-build gcc g++ maven flex bison libxml2-utils cpanminus dpkg-dev
-          cpanm Thread::Pool::Simple
+          sudo apt-get install -yq cmake ninja-build gcc g++ maven flex bison libxml2-utils dpkg-dev
       - name: Configure using CMake
         run: |
           mkdir build
           cd build
           cmake .. -G Ninja -DCMAKE_BUILD_TYPE=Release -DCMAKE_C_COMPILER=/usr/bin/gcc -DCMAKE_CXX_COMPILER=/usr/bin/g++
       - name: Build with Ninja
-        run: cd build; ninja
+        run: cd build; ninja -j2
       - name: Check if package building works
         run: |
           cd build
           ninja package
           ls *.deb
       - name: Run tests
-        run: cd build; ctest . -V -L CORE
+        run: cd build; ctest . -V -L CORE -j2
         env:
           TESTPL_JOBS: 2
```

Comment:
:no_entry_sign: If we are removing the installation of `Thread::Pool::Simple` we should also remove the setting of this environment variable. That will keep things consistent and avoid `test.pl` attempting to use a missing module.